turbo_sort.py

```diff
@@ -63,7 +64,8 @@
 ignore_attributes = ["hdtv", "dts"]
 consolesize = 80
 
-hexaPattern = re.compile('[0-9a-fA-F]+')
+# To avoid problem of false positive hexa, start matching at 6 characters
+hexaPattern = re.compile('[0-9a-fA-F]{6}')
```

Comment:
We can actually do 32 here. Example filename is 7963172165124682a49dc0b70ca2fab1.mkv which is a 256 bit (32 hex character) hash.